example.docker-compose.override.yml

```diff
@@ -0,0 +1,5 @@
+service:
+  environment:
+    AMAZON_S3_KEYID: AmazonS3KeyId
+    AMAZON_S3_KEY: AmazonS3Key
+    UNILOGINSECRET: UniLoginSecret
```

Comment:
Ah, I found the unilogin secret in the configuration:

> Sensitive information that should not be shared outside of the DBC A/S organization.

So this is part of the override.